### DISCUSSION TITLE
Eclipse Theia v1.50.0

### DISCUSSION THREAD

**Discussion by jfaltermeier** [eclipse-theia/theia#13747] [Release Announcements] [answered]
Hi everyone 👋,

The Eclipse Theia v1.50.0 release is scheduled for later today!
We will inform you once the release starts and completes.

**Comment by jfaltermeier** [comment]
The release will start now, I’ll post an update when it has completed.

**Comment by jfaltermeier** [comment]
The release has completed, thank you to everyone that participated and contributed!

**Comment by Jeeppler** [comment]
@jfaltermeier at what time can we expect a new release of TheiaIDE based on Theia v1.50.0?

**Reply by jfaltermeier** [reply]
Hi,

The release process for the Theia IDE is detailed here: https://github.com/eclipse-theia/theia-blueprint/blob/master/PUBLISHING.md#preview-testing-and-release-process-for-the-theia-ide. You can find the thread for preview testing at https://github.com/eclipse-theia/theia/discussions/13781.

The timeline for release could be 1-2 weeks, depending on the number of issues identified during the testing phase. We might also decide to skip a cycle if needed. Feel free to participate in the preview testing and contribute to the process!

**Comment by JonasHelming** [comment, marked as answer]
[News & Noteworthy](https://eclipsesource.com/blogs/2024/06/21/eclipse-theia-1-50-release-news-and-noteworthy/)
